discovery/README.md

```diff
@@ -2,9 +2,8 @@
 
 This directory contains the service discovery (SD) component of Prometheus.
 
-There is currently a moratorium on new service discovery mechanisms being added
-to Prometheus due to a lack of developer capacity. In the meantime `file_sd` 
-remains available.
+Note: new service discovery implementations will be accepted if they have a
```

Comment:
That's only one requirement, we still have all the sanity checks listed below. I'd add this as one more requirement, rather than making it seem like the only requirement.

Reply:
Updated to clarify that it's an additional requirement.